UnitTests/UnitTests.Shared/Mvvm/Test_Messenger.cs

```diff
@@ -550,4 +576,4 @@ public sealed class MessageB
         {
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
@michael-hawker Just a note: this is the same issue you had raised as well while reviewing https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3893: without some automated process to either fix the formatting or at least validate that the state is kept consistent, it's really really easy to reintroduce differences in whitespace formatting while making changes (especially because VS sometimes applies these changes automatically) and still end up with the same situation we had before. As in, the changes in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3893 aren't really that easily maintainable in the long term right now without still ending up with some inconsistencies here and there by accident 🤔